### PR TITLE
feat(azure-resources): Add missing Cognitive Services resources

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Fixes https://github.com/cloudquery/cloudquery/issues/11630

### CODE REVIEW

File: plugins/source/azure/resources/services/cognitiveservices/account_skus.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+)
+
+func accountSKUs() *schema.Table {
+	return &schema.Table{
+		Name:                 "azure_cognitiveservices_account_skus",
+		Resolver:             fetchAccountSKUs,
+		PostResourceResolver: client.LowercaseIDResolver,
+		Description:          "https://learn.microsoft.com/en-us/rest/api/cognitiveservices/accountmanagement/accounts/list-skus?tabs=HTTP#accountsku",
+		Multiplex:            client.SubscriptionMultiplexRegisteredNamespace("azure_cognitiveservices_account_skus", client.Namespacemicrosoft_cognitiveservices),
+		Transform:            transformers.TransformWithStruct(&armcognitiveservices.AccountSKU{}),
```
Comment (erezrokah) [inline plugins/source/azure/resources/services/cognitiveservices/account_skus.go:20]: These don't have an ID. In `azure_compute_skus` we calculate a hash. We could do this here too

File: plugins/source/azure/resources/services/cognitiveservices/account_skus_models.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+)
+
+func accountModels() *schema.Table {
+	return &schema.Table{
+		Name:                 "azure_cognitiveservices_account_models",
+		Resolver:             fetchAccountModels,
+		PostResourceResolver: client.LowercaseIDResolver,
+		Description:          "https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/cognitiveservices/armcognitiveservices@v1.3.0#AccountModel",
+		Multiplex:            client.SubscriptionMultiplexRegisteredNamespace("azure_cognitiveservices_account_models", client.Namespacemicrosoft_cognitiveservices),
+		Transform:            transformers.TransformWithStruct(&armcognitiveservices.AccountModel{}),
```
Comment (erezrokah) [inline plugins/source/azure/resources/services/cognitiveservices/account_skus_models.go:20]: These don't have an ID. In `azure_compute_skus` we calculate a hash. We could do this here too.
Also the models list method is not shown in the REST API docs so I'm linking to the `pkg.go.dev` docs in the description

File: plugins/source/azure/resources/services/cognitiveservices/resource_skus.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+)
+
+func ResourceSKUs() *schema.Table {
+	return &schema.Table{
+		Name:                 "azure_cognitiveservices_resource_skus",
+		Resolver:             fetchResourceSKUs,
+		PostResourceResolver: client.LowercaseIDResolver,
+		Description:          "https://learn.microsoft.com/en-us/rest/api/cognitiveservices/accountmanagement/resource-skus/list?tabs=HTTP#resourcesku",
+		Multiplex:            client.SubscriptionMultiplexRegisteredNamespace("azure_cognitiveservices_resource_skus", client.Namespacemicrosoft_cognitiveservices),
+		Transform:            transformers.TransformWithStruct(&armcognitiveservices.ResourceSKU{}),
```
Comment (erezrokah) [inline plugins/source/azure/resources/services/cognitiveservices/resource_skus.go:19]: these don't have an ID. In `azure_compute_skus` we calculate a hash. We could do this here too.

**Comment by tweaknc** [comment]
Yes that looks great.  I approve.

**Pull request merged**